### PR TITLE
Validator for the number field

### PR DESCRIPTION
**Pull request by jackkum**
Validation session lifetime on client side. (https://github.com/joomla/joomla-cms/issues/5619)

### CODE REVIEW

File: administrator/components/com_config/view/application/html.php [hunk 1]
```diff
@@ -48,6 +48,9 @@ public function render()
 			return false;
 		}
 
+		JHtml::_('behavior.formvalidation');
```
Comment (dgt41) [inline administrator/components/com_config/view/application/html.php]: Can you please use the new version (mootools free)? 
Use: `JHtml::_('behavior.formvalidator');`

**Comment by losedk** [comment]
Would it be possible to add the javascript directly into validate.js?

File: administrator/components/com_config/model/form/application.xml [hunk 2]
```diff
@@ -716,13 +716,15 @@
...
+			type="number"
 			default="15"
 			label="COM_CONFIG_FIELD_SESSION_TIME_LABEL"
 			description="COM_CONFIG_FIELD_SESSION_TIME_DESC"
 			required="true"
 			filter="integer"
-			size="6" />
+			size="6"
+			class="inputbox validate-number"
+			min="5"	/>
```
Comment (mbabker) [inline administrator/components/com_config/model/form/application.xml]: Is there a reason you've hardcoded this to a minimum of 5?  From what I've read, you're just trying to enforce this be a positive value always, so a minimum 1 (even if that really isn't too practical) should be accepted.

Comment (jackkum) [inline administrator/components/com_config/model/form/application.xml]: Totally agree.

File: administrator/components/com_config/view/application/html.php (hunk 1 above)
Comment (jackkum) [inline administrator/components/com_config/view/application/html.php]: Ok. Thanks for the help.

**Comment by bertmert** [comment]
After applying RTC PR https://github.com/joomla/joomla-cms/pull/6294

I've added lines 194 to 201 
https://github.com/bertmert/joomla-cms/blob/patch-3/media/system/js/validate-uncompressed.js#L194

Thus we do not need changes in
administrator/components/com_config/view/application/html.php
and we don't need file
media/com_config/js/validate-number.js
at all.

I made some successful tests.

@jackkum What do you think about these changes.

(((Sorry, I don't know how to manage these changes correctly in github. Multiple files? Minify? Just know how to create a new PR with a single file.)))

**Comment by jackkum** [comment]
Sorry, not enough time to peek here :baby: 
@bertmert I think it can be usefull. 
Fedik proposed this https://gist.github.com/phproberto/6933178 
Just modify file validate-uncompressed.js and compress him in validate.js
```
git checkout -b number-validator
git add validate-uncompressed.js
git add validate.js 
git commit -m "some desc"
git pull origin number-validator
```
and create a PR

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on b0210d141b6c9db9808bea184f9e6c129c720686<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/5624">issues.joomla.org/joomla-cms/5624</a>.</sub>

**Comment by roland-d** [comment]
@jackkum Could you please solve the merge conflicts, add a minified version of the JS file and give us an instruction on how to test this? Thanks.

**Comment by jackkum** [comment]
@roland-d Ok.

**Comment by roland-d** [comment]
@jackkum Thank you very much.

**Comment by jackkum** [comment]
@roland-d 
#9997 this is for validation only in com_config. There have some trouble with tests, i don't understand what is wrong :confused: 
#9998 this is for global validation handler 
Choose one. :smiley: 
Close this PR?

**Comment by Bakual** [comment]
@jackkum It's only a codestyle issue in that PR. See https://travis-ci.org/joomla/joomla-cms/jobs/124358513 at the bottom:
````
FILE: ...oomla-cms/administrator/components/com_config/view/application/html.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 52 | ERROR | Tabs must be used to indent lines; spaces are not allowed
````

**Comment by jackkum** [comment]
@Bakual Thanks.

**Comment by brianteeman** [comment]
Please see #9998<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/5624">issues.joomla.org/joomla-cms/5624</a>.</sub>

**Pull request closed without merging**